modules/op-release-notes-1-8.adoc

```diff
@@ -36,22 +282,179 @@ For the *Tekton Operator*, the `operator.tekton.dev/v1alpha1` API version is *no
 ====
 //  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
 
-
 * In {pipelines-title} 1.8, the `PipelineResource` custom resource (CR) is available but no longer supported. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API, which had been deprecated and planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
 
 * In {pipelines-title} 1.8, the `Condition` custom resource (CR) is removed. The `Condition` CR was part of the `tekton.dev/v1alpha1` API, which has been deprecated and is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
 
+* In {pipelines-title} 1.8, the `gcr.io` image for `gsutil` has been removed. This removal might break clusters with `Pipeline` resources that depend on this image. Bug fixes and support are provided only through the end of the {pipelines-title} 1.7 lifecycle.
+// link:https://issues.redhat.com/browse/SRVKP-1539
+// Shivam Mukhade
+
+// .Pipelines
+
+* In {pipelines-title} 1.8, the `PipelineRun.Status.TaskRuns` and `PipelineRun.Status.Runs` fields are deprecated and are planned to be removed in a future release. See link:https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md[TEP-0100: Embedded TaskRuns and Runs Status in PipelineRuns].
+
+* In {pipelines-title} 1.8, the `pipelineRunCancelled` state is deprecated and planned to be be removed in {pipelines-title} 1.9. Graceful termination of `PipelineRun` objects is now promoted from being an alpha feature to being a stable feature. (See link:https://github.com/tektoncd/community/blob/main/teps/0058-graceful-pipeline-run-termination.md[TEP-0058: Graceful Pipeline Run Termination].) As an alternative, you can use the `Cancelled` state, which replaces the `pipelineRunCancelled` state.
```

Comment:
```suggestion
* In {pipelines-title} 1.8, the `pipelineRunCancelled` state is deprecated and planned to be be removed in a future release. Graceful termination of `PipelineRun` objects is now promoted from an alpha feature to a stable feature. (See link:https://github.com/tektoncd/community/blob/main/teps/0058-graceful-pipeline-run-termination.md[TEP-0058: Graceful Pipeline Run Termination].) As an alternative, you can use the `Cancelled` state, which replaces the `pipelineRunCancelled` state.
```
Shouldn't we avoid specifying a version in which it will be removed?

Reply:
Updated. Thanks.